libs/sdk-ui-dashboard/src/model/events/layout.ts

```diff
@@ -43,6 +53,7 @@ export function layoutSectionAdded(
     ctx: DashboardContext,
     section: ExtendedDashboardLayoutSection,
     index: number,
+    path: ILayoutSectionPath,
```

Comment:
Similar story to command builders but in this case TypeScript overload is not used as these events are build only internally and the builders are not exposed outside of the libs. The refactor of the function signatures was therefore safe.